jvm-runtime/ftl-runtime/common/deployment/src/main/java/xyz/block/ftl/deployment/ModuleBuilder.java

```diff
@@ -82,9 +82,11 @@ public class ModuleBuilder {
     private final Map<DotName, TopicsBuildItem.DiscoveredTopic> knownTopics;
     private final Map<DotName, VerbClientBuildItem.DiscoveredClients> verbClients;
     private final FTLRecorder recorder;
+    private final Map<String, String> verbDocs;
```

Comment:
Just to make sure you know, most declarations in the FTL schema can include comments - data structures, enums, etc.